<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessFinder</class>
 <widget class="QDialog" name="ProcessFinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProcessFinder</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QScrollArea {
	border:none;
	background:transparent;
}

QScrollBar:vertical {
	background:transparent;
	padding:0px;
	width:10px;
}

QScrollBar::handle:vertical{
	background:rgba(0,0,0,35);
	border-radius:5px;
	min-height:30px;
}

QScrollBar::handle:vertical:hover,QScrollBar::handle:vertical:pressed {
	background:rgba(0,0,0,75);
}

QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical {
	background:none;
}

QScrollBar::add-line:vertical,QScrollBar::sub-line:vertical {
	background:none;
}

QRadioButton
{
    spacing: 5px;
	font-size:15px;
}

QRadioButton::indicator
{
    width: 16px;
    height: 16px;
}

QRadioButton::indicator:unchecked
{
	image: url(:/AnimismAI/Resource/checkbox-unchecked.png);
}

QRadioButton::indicator:checked
{
	image: url(:/AnimismAI/Resource/checkbox-indeterminate.png);
}

QCheckBox
{
    spacing: 5px;
	font-size:15px;
}

QCheckBox::indicator
{
    width: 16px;
    height: 16px;
}

QCheckBox::indicator:unchecked
{
	image: url(:/AnimismAI/Resource/checkbox-unchecked.png);
}

QCheckBox::indicator:checked
{
	image: url(:/AnimismAI/Resource/checkbox-checked.png);
}

QSpinBox
{
	font-size: 13px;
	border: 1px solid rgb(160,160,160);
	background-color: transparent;
	padding-left: 5px;
	padding-right: 5px;
}

QTextEdit
{
	font-size: 12px;
	border: 1px solid rgb(160,160,160);
	background-color: transparent;
}

QListWidget
{
	font-size: 12px;
	border: 1px solid rgb(160,160,160);
	background-color: transparent;
}

QLineEdit
{
	font-size: 12px;
	border: 1px solid rgb(160,160,160);
	background-color: transparent;
}

QTreeWidget
{
	font-size: 12px;
	border: 1px solid rgb(160,160,160);
	background-color: transparent;
}

QTreeWidget::item 
{
    spacing: 5px;
	font-size:15px;
}

QTreeWidget::indicator
{
    width: 16px;
    height: 16px;
}

QTreeWidget::indicator:unchecked
{
	image: url(:/AnimismAI/Resource/checkbox-unchecked.png);
}

QTreeWidget::indicator:checked
{
	image: url(:/AnimismAI/Resource/checkbox-checked.png);
}

QTreeWidget::indicator:indeterminate
{
	image: url(:/AnimismAI/Resource/checkbox-indeterminate.png);
}

QListWidget
{
	font-size: 12px;
	border: 1px solid rgb(160,160,160);
	background-color: transparent;
}

QLineEdit
{
	font-size: 12px;
	border: 1px solid rgb(160,160,160);
	background-color: transparent;
}

QPushButton{font-size:12px;}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_frame" native="true">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>360</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>360</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_app_head" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>7</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_app_icon" native="true">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>727</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="WinButton" name="pushButton_app_close">
           <property name="minimumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="toolTip">
            <string>关闭</string>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_container" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEdit_find">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>查找目标进程</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_proc">
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>178</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="IconButton" name="pushButton_refresh">
             <property name="minimumSize">
              <size>
               <width>65</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>65</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>刷新</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="IconButton" name="pushButton_submit">
             <property name="minimumSize">
              <size>
               <width>65</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>65</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>确认</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="IconButton" name="pushButton_cancel">
             <property name="minimumSize">
              <size>
               <width>65</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>65</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>取消</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="StatusBar" name="widget_app_status" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WinButton</class>
   <extends>QPushButton</extends>
   <header>WinButton.h</header>
  </customwidget>
  <customwidget>
   <class>StatusBar</class>
   <extends>QWidget</extends>
   <header>StatusBar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IconButton</class>
   <extends>QPushButton</extends>
   <header>IconButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
